src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/integration/resource/JobTriggerIntegrationTest.kt

```diff
@@ -152,14 +152,17 @@ class JobTriggerIntegrationTest : IntegrationTestBase() {
   }
 
   @Test
-  @Sql("classpath:integration-test-data/seed-migrated-bookings.sql")
+  @Sql("classpath:test_data/clean-all-data.sql", "classpath:integration-test-data/seed-migrated-bookings.sql")
   fun `should not send an email to the court to remind them to add the court hearing link if the booking has been migrated`() {
-    notificationRepository.findAll() hasSize 0
+    assumingThat(today().dayOfWeek in listOf(MONDAY, TUESDAY, WEDNESDAY, THURSDAY, SUNDAY)) {
+      notificationRepository.findAll() hasSize 0
```

Comment:
Data was hanging around causing the first assert to fail.

Not 100% sure why added the clean specific to the endpoint to work around this for now.  Will come back to it.